sys-utils/flock.c

```diff
@@ -237,6 +240,7 @@ int main(int argc, char *argv[])
 	bindtextdomain(PACKAGE, LOCALEDIR);
 	textdomain(PACKAGE);
 	close_stdout_atexit();
+	shell = ul_default_shell(0, NULL); /* Used for -c command execution */
 
```

Comment:
The shell use is optional. It would be better to move this to `/* Run command */` code block, rather than requesting the shell path for all time.